src/opf/utils.jl

```diff
@@ -0,0 +1,301 @@
+using MathOptSymbolicAD
+using SparseArrays
+
+struct OPFData
```

Comment:
Please move `OPFData` data structure to `src/opf/opf.jl`

Keep wr/wi bounds etc in `utils.jl` here